### PR TITLE
Add href to page title

### PR DESCRIPTION
**Pull request by ski7777**
Looks exactly like the original one but it is clickable

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
           </a>
         </div>
       </div>
-      <a href="{{ site.baseurl }}/{{ lang }}/">
+      <a href="{{ site.baseurl }}/{{ lang }}/" style="color: #ffffff; text-decoration: none;">
```
Comment (rkunze) [inline docs/_layouts/default.html]: Please change the style in css/screen.scss instead of inline in the HTML template.

**Comment by ski7777** [comment]
This way?

File: docs/_layouts/default.html (hunk 1 above)
Comment (ski7777) [reply docs/_layouts/default.html]: Done 1f37e703

**Comment by rkunze** [comment]
Yes, fine.

**Pull request merged**